docs/README.md

```diff
@@ -2,32 +2,55 @@
 navTitle: Documentation
 ---
 
-# FlowForge Docs
+# Welcome to FlowForge
 
 FlowForge helps Node-RED developers deliver applications in a more reliable,
 collaborative, and secure manner. Node-RED’s intuitive, low-code development
 environment is great for connecting together hardware devices, APIs, and online
-services. FlowForge adds to Node-RED; collaborative development, management of
+services.
+
+FlowForge adds to Node-RED; collaborative development, management of
 remote deployments, support for DevOps delivery pipelines, and the ability to
 host Node-RED applications on FlowForge. FlowForge is the DevOps platform for
 Node-RED application development and delivery.
 
-## Overview
+## Using FlowForge
```

Comment:
Is this first time users we're targetting?

```suggestion
## Getting started
```

Reply:
Yep, that's fair.